tests/testthat/test-g_mmrm.R

```diff
@@ -24,16 +24,37 @@ fit_mmrm_no_arms_object <- fit_mmrm(
 # g_mmrm_diagnostic ----
 
 test_that("g_mmrm_diagnostic works well with defaults", {
+  result <- g_mmrm_diagnostic(fit_mmrm_object)
+  expect_silent(result)
+})
+
+test_that("g_mmrm_diagnostic plot works well with defaults", {
+  skip_on_ci()
```

Comment:
instead of replicating the whole test, could you maybe just put `skip_on_ci()` just before the `expect_doppelganger()` call?

Reply:
Updated - this seems to be working now